java/gradle.properties

```diff
@@ -0,0 +1,10 @@
+#org.gradle.configuration-cache=true
+org.gradle.parallel=true
+org.gradle.caching=true
+
+# extra JVM args needed to allow Spotless to access JDK private classes.
+org.gradle.jvmargs =--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED \
```

Comment:
❤️ nothing better than `--add-exports`

Reply:
I have even more --add-exports now for spark :D